examples/mutual-ssl/mutual_ssl_service.bal

```diff
@@ -13,7 +13,7 @@ http:ListenerConfiguration helloWorldEPConfig = {
             path: "${ballerina.home}/bre/security/ballerinaTruststore.p12",
             password: "ballerina"
         },
-         // Configure the preferred SSL protocol and the versions to enable.
+         // Configure the preferred SSL protocol and the versions to enable the HTTP Listener.
```

Comment:
It's not "HTTP Listener". @Bhashinee what's the correct term to use here? What are we enabling by setting these configs? HTTPS?

Reply:
Changed to "Enable the preferred SSL protocol and its versions.".